Makefile

```diff
@@ -1,55 +1,74 @@
 .DEFAULT_GOAL=all
 
-# Formatting
+.PHONY: help
+help: ## Print this help
+	@printf -- '\n'
+	@printf -- '                                                     __ \n'
+	@printf -- '             (≡)         ____ _ _____ ____ _ ____   / / \n'
+	@printf -- '                \       / __ `// ___// __ `// __ \ / /  \n'
+	@printf -- '                (≡)    / /_/ // /   / /_/ // /_/ // /   \n'
+	@printf -- '                /      \__, //_/    \__,_// .___//_/    \n'
+	@printf -- '             (≡)      /____/             /_/            \n'
+	@printf -- '\n'
```

Comment:
Not sure we need the Grapl ASCII art for this, given that it's not the name of the project at hand.

Reply:
Yeah I went back and forth about it since it is part of the grapl repo. I think its fair to remove it though